content/components/comment-box.mdx

```diff
@@ -0,0 +1,199 @@
+---
+title: Comment box
+status: Experimental
+---
+
+import {Box, Label, LabelGroup, Text} from '@primer/react'
+
+<Box sx={{fontSize: 3}} class="lead" as="p">
+  A comment box allows users to write and preview comments, format Markdown, autocomplete user names or references and attach files.
+</Box>
+
+<!-- TODO: uncomment when available
+<LabelGroup>
+  <Label
+    as="a"
+    href="https://primer.style/react/CommentBox"
+    size="large"
+    variant="secondary"
+    sx={{textDecoration: 'none'}}
+  >
+    <img
+      width="16"
+      height="16"
+      alt=""
+      src="https://user-images.githubusercontent.com/293280/123878374-ce9d4d00-d8f3-11eb-8adf-1a160292ff53.png"
+      style="vertical-align: middle; margin-right: 4px;"
+    />
+    React
+  </Label>
+  <Label
+    as="a"
+    href=""
+    size="large"
+    variant="secondary"
+    sx={{textDecoration: 'none'}}
+  >
+    <img
+      width="16"
+      height="16"
+      alt=""
+      src="https://user-images.githubusercontent.com/293280/123878720-80d51480-d8f4-11eb-9b10-d02a1cb606f8.png"
+      style="vertical-align: middle; margin-right: 4px;"
+    />
+    Figma
+  </Label>
+  <Label
+    as="a"
+    href="https://primer.style/view-components/components/alpha/CommentBox"
+    size="large"
+    variant="secondary"
+    sx={{textDecoration: 'none'}}
+  >
+    <img
+      width="16"
+      height="16"
+      alt=""
+      src="https://user-images.githubusercontent.com/18661030/171468311-184bfd70-128e-4bfc-b482-96d0de49d42a.png"
+      style="vertical-align: middle; margin-right: 4px;"
+    />
+    Rails
+  </Label>
+</LabelGroup>
+-->
+
+## Anatomy
+
+<img width="960" alt="A diagram of an comment box with a label, a TabNav, an input and an ActionBar." src="https://user-images.githubusercontent.com/378023/191439493-2122cae7-c03a-4c8b-b583-1919ccb34d94.png" />
+
+## Content
+
+### Label
+
+A label is shown above the comment box to inform users what kind of text input is following. The default label is "Add a comment".
+
+<DoDontContainer>
+  <Do>
+    <img
+      src="https://user-images.githubusercontent.com/378023/191459919-0301444a-18cc-474e-8db7-1480976e4128.png"
+      role="presentation"
+      width="456"
+    />
+    <Caption>The label can be changed depending on the context.</Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://user-images.githubusercontent.com/378023/191459943-1bc24fb4-224e-4598-94b6-f15b80401946.png"
+      role="presentation"
+      width="456"
+    />
+    <Caption>Don't use the label to notify users of any kind of state or event, like when an error occurred.</Caption>
+  </Dont>
+</DoDontContainer>
+
+### Placeholder
+
+In most cases a placeholder text is not needed and the label will suffice.
```

Comment:
Hey @simurai I am just curious, is this suggested by the a11y team? Personally I feel that without the placeholder it does not look so much like I could put text into it. But this may also be my personal preference.

Reply:
It's how I understood it, but might not be accurate. A proper accessibility review will still follow. I also have a hard time coming up with a good placeholder text when there is already a label above the comment box. The example in this guide says "Type here...", but that sounds a bit too blunt? 🤔

<img src="https://user-images.githubusercontent.com/378023/191459984-243b1223-5029-4915-afc9-53b89e6e8ce3.png" alt="comment box show a placeholder text that reads: Type here..." width="456" />

But I think I know what you mean, a placeholder text is good for usability as a hint that it's an input.